cmake/Modules/GrPython.cmake

```diff
@@ -33,8 +33,7 @@ if (PYTHON_EXECUTABLE)
     find_package(PythonInterp ${GR_PYTHON_MIN_VERSION} REQUIRED)
 else (PYTHON_EXECUTABLE)
     message(STATUS "PYTHON_EXECUTABLE not set - using default python3")
-    message(STATUS "Use -DPYTHON_EXECUTABLE=/path/to/python2 to build for python2.")
-    find_package(PythonInterp ${GR_PYTHON3_MIN_VERSION} REQUIRED)
+    find_package(PythonInterp ${GR_PYTHON_MIN_VERSION} REQUIRED)
```

Comment:
OK so maybe this is a change for another PR ... but this "if" statement makes no sense now! It's doing a different status message then the same search. There has -got- to be a better way!

Reply:
looks like this would work ... haven't tested it yet though: `find_package(Python3 COMPONENTS Interpreter Development)`

Reply:
uuuuuh I like that!

Reply:
Note that the variables returned by this command are `PYTHON3_*` ... so we'd want to also set `PYTHON_*` from them. Small price to pay ...

Reply:
I *have* tested for UHD, and for CMake 3.8, this won't work. It needs a newer version. I had to make a stupid compat layer in UHD to support both :angry:

Reply:
@mbr0wn ?meaning? ... not sure how this connects to the comment thread here ...

Reply:
find_package(Python3) needs a version of CMake that we don't require.

Reply:
You're right on both accounts: This can be improved, and it can be done in another commit.